gitea/release.go

```diff
@@ -48,6 +49,33 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAttachments gets all the assets of a release in a repository
+func (c *Client) ListReleaseAttachments(user, repo string, id int64) ([]*Attachment, error) {
+	attachments := make([]*Attachment, 0, 10)
+	err := c.getParsedResponse("GET",
+		fmt.Sprintf("/repos/%s/%s/releases/%d/assets", user, repo, id),
+		nil, nil, &attachments)
+	return attachments, err
+}
+
+// GetReleaseAttachment gets a single attachment of a release in a repository
+func (c *Client) GetReleaseAttachment(user, repo string, releaseID int64, attachmentID int64) (*Attachment, error) {
+	attachment := new(Attachment)
+	err := c.getParsedResponse("GET",
+		fmt.Sprintf("/repos/%s/%s/releases/%d/assets/%d", user, repo, releaseID, attachmentID),
```

Comment:
This route is incorrect: https://developer.github.com/v3/repos/releases/#get-a-single-release-asset

Reply:
It is incorrect, but the modeling of github is different from what gitea has.
In github, as you can see, you don't specify the release id. Assets are associated with the releases of a repo (not to a specific one). In gitea, the mapping between an attachment and a repo is done through a specific release. Also, we are never referencing this url in other endpoints. In github, ListReleases also has links to asset metadata. We don't have something like that.
I think the best thing is to just remove this endpoint. Or, of course i welcome other suggestions.

Reply:
I think we could keep `v1` not compitable with github but `v3` comiptable.

Reply:
@stefan-lacatus AFAIK, an asset is uniquely identified by its id. Although an attachment is associated with a particular issue or release, you do not need which issue or release in order to identify an asset by its id. So, as far as I can tell, supporting the Github-style route should not be a problem.